generators/entity-server/templates/src/main/java/package/service/EntityQueryService.java.ejs

```diff
@@ -95,7 +97,9 @@ public class <%= serviceClassName %> extends QueryService<<%= persistClass %>> {
      */
     @Transactional(readOnly = true)
     public Page<<%= instanceType %>> findByCriteria(<%= criteria %> criteria, Pageable page) {
-        log.debug("find by criteria : {}, page: {}", criteria.toString().replaceAll("[\n\r\t]", "_"), page);
+        if (log.isDebugEnabled()) {
+            log.debug("find by criteria : {}, page: {}", criteria.toString().replaceAll("[\n\r\t]", "_"), page);
+        }
```

Comment:
page is missing the replacement too.
https://sonarcloud.io/project/issues?resolved=false&types=VULNERABILITY&id=jhipster-sample-application&open=AXk8UlfKmxD5nLr49ZDM